readme.md

```diff
@@ -1 +1,73 @@
+# Repositories and packages naming conventions
 
+*All new public repositories and packages should follow naming conventions, we don't want to rename existing projects. Provided examples are for visualizing all currently available repositories with the proposed naming conventions.*
+
+## Repositories
+
+*The main idea in the proposed naming convention is to provide easy readability and information to the developers. Moreover, it will help us make the organization consistent. In most cases, the developer is interested in the repositories belonging to a specific product, therefore, the repository name starts with the product name. Then developer is looking for the specific project for his/her needs for the specific platform. Therefore `<project-name>-<language/platform>` suffix.*
+
+---
+
+### Is the repository related to the company's product?
+
+#### Format: `<product>-<project-name>-<language/platform>`
+
+- `<product>` - mandatory, related product
+  - *Examples: `fingerprintjs`, `fingerprintjs-pro`, `botd`*
+- `<project-name>` - mandatory, descriptive name of the project
+  - *Examples: `server-api-sdk`, `agent`, `client`, `integrations`*
+- `<language/platform>` - optional, use when makes sense or it's plausible there will be available the same functionality for different language/platform in the future. Mostly apply to agents, wrappers, or SDKs.
+  - *Examples: `javascript`, `node`, `react`, `python`*
+
+#### Examples
+
+- `fingerprintjs-agent` or `fingerprintjs-client`
+- `botd-agent` or `botd-client`
+- `botd-integrations`
+- `fingerprint-android-client` or `fingerprintjs-android-client`
```

Comment:
Shouldn't it be `fingerprint-client-android` because `client` is `<project-name>` and `android` is `<language/platform>`?

Reply:
“Fingerprint Android” is a product name as far as I understand. That is, it isn’t an Android integration for the “FingerprintJS” product, it’s a separate product.

Reply:
@Finesse  I like your point of view^. Therefore:
- product: `fingerprint-android`
- project name: `client` ?
- language/platform: `kotlin`
Result: `fingerprint-android-client-kotlin`

Your thoughts @Finesse @ilya-nurullin?

Reply:
I'm not sure whether there can be clients for different platforms and languages. @Alexey-Verkhovsky, what do you think?

Reply:
I think we can easily omit platform/language here

Reply:
I agree with all of you

Reply:
thx guys, I'll incorporate it to the kb